plugins/qlcplus.js

```diff
@@ -167,11 +163,33 @@ function exportHandleAvailableChannels(fixture, defaults) {
         }
       }
     }
+
+    if ('fineChannelAliases' in chData) {
+      chData.fineChannelAliases.forEach((alias, index) => {
+        let fineXmlChannel = JSON.parse(JSON.stringify(xmlChannel));
+
+        fineXmlChannel.$.Name += getFineChannelSuffix(index);
```

Comment:
Is the uniqueness of this channel name guaranteed? Why do we actually need to add the fine suffix if there already is a unique fine channel key specified?